test/unit/algorithm_mode/test_custom_metrics.py

```diff
@@ -44,6 +44,7 @@ def test_mse():
 multiclass_train_label = np.array([0, 0, 1, 1, 1, 1, 1, 2, 2, 2])
 multiclass_dtrain = xgb.DMatrix(multiclass_train_data, label=multiclass_train_label)
 multiclass_preds = np.ones(10)
+multiclass_preds_softprob = np.asarray([[0.8, 0.1, 0.1]] * 10)
```

Comment:
Please take the array with different values in each row, to ensure that behaving as expected.

Reply:
Sure, updated the values.